pkg/manifest/os.go

```diff
@@ -406,6 +406,7 @@ func (p *OS) serialize() osbuild.Pipeline {
 
 	if len(p.containerSpecs) > 0 {
```

Comment:
I was thinking about this a little more. Maybe the container specs could be a map rather than a slice? And the key could be the source name? Just one idea of how to simplify this